target.json

```diff
@@ -2,7 +2,7 @@
   "name": "nrf51dk-armcc",
   "version": "0.0.8",
   "inherits": {
-    "nordic-nrf51822-32k-armcc": "*"
+    "nordic-nrf51822-armcc": "*"
```

Comment:
^0.0.0 would mean take an 0.x.y. When this target changes its major to 1.0.0, dependent targets (leaf targets) may need to be isolated from the major change. This is the same argument we use for versioning elsewhere.